src/collective/ifttt/interfaces.py

```diff
@@ -1,8 +1,23 @@
 # -*- coding: utf-8 -*-
 """Module where all interfaces, events and exceptions live."""
 
+from collective.ifttt import _
+from plone.autoform import directives
+from zope import schema
+from zope.interface import Interface
 from zope.publisher.interfaces.browser import IDefaultBrowserLayer
 
 
 class ICollectiveIftttLayer(IDefaultBrowserLayer):
     """Marker interface that defines a browser layer."""
+
+
+class IIftttControlPanel(Interface):
```

Comment:
@Shriyanshagro One more. It's convention that this interfaces describing control panel settings is named as *Settings.

Like ``IIftttSettings`` or ``ICollectiveIFtttSettings``.

That's because those settings maybe later looked up from Plone settings registry with that interface name.

Reply:
done